projects/kpt.dev/package.yml

```diff
@@ -0,0 +1,31 @@
+# https://kpt.dev/
+
+distributable: ~
+warnings:
+  - vendored
+
+versions:
+  github: kptdev/kpt/tags
```

Comment:
I couldn't make it to install latest beta version. How can I @jhheider @mxcl?

```
pkg build kpt.dev@v1.0.0-beta.46
error: Uncaught Error: invalid semver range: @v1.0.0-beta.46
      const err = () => new Error(`invalid semver range: ${input}`)
                        ^
    at err (https://deno.land/x/libpkgx@v0.14.1/src/utils/semver.ts:137:25)
    at new Range (https://deno.land/x/libpkgx@v0.14.1/src/utils/semver.ts:196:49)
    at Module.parse (https://deno.land/x/libpkgx@v0.14.1/src/utils/pkg.ts:11:22)
    at file:///Users/vaibhavkumbhar/.pkgx/pkgx.sh/brewkit/v0.49.0/libexec/find.ts:8:43
```

Reply:
at present, we only take semver versions without build or pre-release identifiers. we haven't yet had a clamor to add this, but it could be possible.

Reply:
We haven't yet decided how to handle betas since we haven't had a good example of one. Is this a good example of one we should support? @vkumbhar94

Reply:
Sounds like a good example as in the last stable version 0.39.3 was released around 2 yrs ago.
Since then v1.X.X beta versions; last beta released just 5 days ago.
I am just thinking out loud that it is going through some major redesign so they haven't yet called out a stable v1, they are just continuing beta.
But if we look at it from the user's point of view, there would be several features made available in beta since the last 2 yrs. Thus one may tend to use the beta version rather than using stable version released a few yrs back.